src/main/java/at/laborg/briss/BrissSwingGUI.java

```diff
@@ -508,19 +548,18 @@ public void setDefinedSizeSelRects() {
         if ((maxWidth >= 0) && (maxHeight >= 0)) {
             maxWidth = Math.round(25.4f * maxWidth / 72f);
             maxHeight = Math.round(25.4f * maxHeight / 72f);
-            defInput = maxWidth + " " + maxHeight; //$NON-NLS-1$
+            defInput = maxWidth + " " + maxHeight; // $NON-NLS-1$
         }
 
         // get user input
         // maximums are used as a default
         String input = JOptionPane.showInputDialog(Messages.getString("BrissGUI.size"), defInput);
 
-        if (input == null || input.equals("")) //$NON-NLS-1$
-            return;
+        if (input == null || input.equals("")) // $NON-NLS-1$
+        return;
```

Comment:
This formatting seems to be wrong?

Reply:
That's weird. I'll make sure to double check it and use curly braces.